css/style.css

```diff
@@ -30,7 +64,6 @@ a:focus {
 }
 h2 {
 	font-size: 32px;
-	font-weight: 300;
```

Comment:
Sorry to backtrack, but looking at the website, I think I prefer it without the boldness. Mind using `font-weight: normal` instead? Super minor nitpick so don't worry about it if you're bothered. I'll approve either way and you do what you want.

Reply:
Didn't realize it defaulted to bold, will change